modules/prelude-go.el

```diff
@@ -31,10 +31,13 @@
 (require 'prelude-programming)
 
 (prelude-require-packages '(go-mode
-                            company-go
-                            go-eldoc
-                            go-projectile
-                            gotest))
+                            lsp-mode
+                            lsp-ui
+                            company
+                            yasnippet
```

Comment:
Any special reason for adding `yas` just for go?

Reply:
Removed. I'll add it in the documentation which I'm about to write; `go` is highly redundant which makes yas very useful but this belongs in a doc, not automatically loaded.